src/nvim/mark.c

```diff
@@ -1204,16 +1246,27 @@ void cleanup_jumplist(win_T *wp, bool checktail)
         break;
       }
     }
-    if (i >= wp->w_jumplistlen) {  // no duplicate
+    bool shouldFree = false;
```

Comment:
nit: this is typically called `mustfree` elsewhere in the codebase. Also  snake_case, not camelCase (except for type names).

Reply:
Fixed in https://github.com/neovim/neovim/pull/11530/commits/3aca2ea5bcaf9cbc1ad7eb93ca98176e753cbfd4 .